package.json

```diff
@@ -19,6 +19,7 @@
   "license": "MIT",
   "dependencies": {
     "@babel/polyfill": "^7.4.4",
+    "@sendgrid/mail": "^6.4.0",
```

Comment:
Why do you have sendgrid and node mailer at the same time, can you stick to using one of it?

Reply:
fixed